chalicelib/services/BroadcastService.py

```diff
@@ -57,35 +78,215 @@ def generate_newsletter_content(self, custom_content: str = "") -> Dict:
+                text-align: center;
+            }
+            
+            .newsletter-subtitle {
+                font-size: 20px;
+                text-transform: uppercase;
+                letter-spacing: 1px;
+                border-top: 1px solid #888;
+                border-bottom: 1px solid #888;
+                padding: 5px 0;
+                margin: 10px 0 20px;
+                text-align: center;
+            }
+            
+            .welcome-message {
+                padding: 20px;
+                background-color: #fff;
+                text-align: center;
+            }
+            
+            .main-content {
+                padding: 0 20px;
+            }
+            
+            .section-banner {
+                width: 100%;
+                margin: 0 0 15px 0;
+                overflow: hidden;
+            }
+            
+            .section-banner img {
+                width: 100%;
+                display: block;
+            }
+            
+            .job-opportunities-banner {
+                width: 100%;
+                margin: 20px 0 0 0;
+                overflow: hidden;
+            }
+            
+            .job-opportunities-banner img {
+                width: 100%;
+                display: block;
+            }
+            
+            .job-sources {
+                background-color: #f9f9f9;
+                padding: 15px;
+                margin: 0 0 20px 0;
+                font-size: 14px;
+                line-height: 1.5;
+                border-bottom: 1px solid #eee;
+            }
+            
+            .section-title {
+                font-size: 28px;
+                text-transform: uppercase;
+                text-align: center;
+                background-color: #f2f2f2;
+                padding: 15px;
+                margin: 0 0 20px;
+                border: none;
+            }
+            
+            .jobs-section {
+                margin-bottom: 40px;
+            }
+            
+            .job-item {
+                margin-bottom: 20px;
+                padding: 0 0 15px 0;
+                border-bottom: 1px dotted #ddd;
+            }
+            
+            .job-title {
+                font-weight: bold;
+                margin-bottom: 5px;
+            }
+            
+            .job-details {
+                font-size: 14px;
+            }
+            
+            .events-section {
+                margin-bottom: 40px;
+            }
+            
+            .footer {
+                text-align: center;
+                padding: 20px;
+                border-top: 1px solid #ddd;
+            }
+            
+            .social-links {
+                text-align: center;
+                margin: 20px 0;
+            }
+            
+            .social-links a {
+                margin: 0 10px;
+                text-decoration: none;
+            }
         </style>
         """
         
+        # Job sources text
```

Comment:
What if the job sources change in the long run? I think it'd be easier to maintain if we had a const tracking the links and names, something along the lines of:
```
JOB_SOURCE_LIST = {
  "tech": [{name: "...", link: "..."}, ...],
  "finance": [...],
  ...
}
```
and you can iterate through it and embed as an fstring. If it's too complicated, feel free to leave it be.